src/modules/_nomodule/PixelManipulation.js

```diff
@@ -3,84 +3,84 @@
 * accepting a changePixel() method to remix a pixel's channels
 */
 module.exports = function PixelManipulation(image, options) {
-  
   options = options || {};
-  options.changePixel = options.changePixel || function changePixel(r, g, b, a) {
-    return [r, g, b, a];
-  };
+  options.changePixel =
+    options.changePixel ||
+    function changePixel(r, g, b, a) {
+      return [r, g, b, a];
+    };
+
+  //
+  options.extraManipulation =
+    options.extraManipulation ||
+    function extraManipulation(pixels) {
+      return pixels;
+    };
+
+  var getPixels = require("get-pixels"),
+    savePixels = require("save-pixels");
 
-  // 
-  options.extraManipulation = options.extraManipulation || function extraManipulation(pixels){
-    return pixels;
-  }
-  
-  var getPixels = require('get-pixels'),
-  savePixels = require('save-pixels');
-  
   getPixels(image.src, function(err, pixels) {
-    
-    if(err) {
-      console.log('Bad image path', image);
+    if (err) {
+      console.log("Bad image path", image);
       return;
     }
 
-    if(options.getNeighbourPixel){
-      options.getNeighbourPixel.fun = function getNeighborPixel(distX,distY) { 
-        return options.getNeighbourPixel(pixels,x,y,distX,distY);
+    if (options.getNeighbourPixel) {
+      options.getNeighbourPixel.fun = function getNeighborPixel(distX, distY) {
+        return options.getNeighbourPixel(pixels, x, y, distX, distY);
       };
     }
-    
+
     // iterate through pixels;
     // TODO: this could possibly be more efficient; see
     // https://github.com/p-v-o-s/infragram-js/blob/master/public/infragram.js#L173-L181
-    
-    if (!options.inBrowser) {
+
+    if (!options.inBrowser && !process.env.TEST) {
```

Comment:
@jywarren

Reply:
Perfect, that's all I needed. Merging now!